cpp-client/deephaven/client/src/highlevel/impl/table_handle_impl.cc

```diff
@@ -65,11 +65,14 @@ using io::deephaven::barrage::flatbuf::ColumnConversionMode;
 using io::deephaven::barrage::flatbuf::CreateBarrageMessageWrapper;
 using io::deephaven::barrage::flatbuf::CreateBarrageSubscriptionOptions;
 using io::deephaven::barrage::flatbuf::CreateBarrageSubscriptionRequest;
-using io::deephaven::barrage::flatbuf::GetBarrageMessageWrapper;
 
 namespace deephaven::client::highlevel {
 namespace impl {
 
+inline const io::deephaven::barrage::flatbuf::BarrageMessageWrapper *GetBarrageMessageWrapper(const void *buf) {
```

Comment:
Super-nit: as a style thing I avoid the `inline` keyword unless it's required (like in a .h file where a function definition would otherwise be multiply included). Here it feels a little noisy. The compiler can choose to inline this function or not -- it doesn't really need our help -- and if it does not choose to inline, then it's not worth much as a compiler.

Up to you.

Reply:
Fixed; it wasn't even intentional, I was not thinking, it was just a cut&paste from the generated code in the header with the added line.